src/TableHeaderCells.js

```diff
@@ -0,0 +1,67 @@
+import _ from "lodash"
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Table } from 'evergreen-ui'
+import styled from 'styled-components'
+
+const TableSearchHeaderCellDiv = styled.div`
+ display: ruby;
+`
+
+function renderTableSortHeaderCell (columnNumber, headerText, width, orderedColumn, ordering, onClick) {
+  return <Table.TextHeaderCell
+    key={`header${columnNumber}`}
+    className={`header${columnNumber}`}
+    flexBasis={width} flexShrink={100} flexGrow={100}
+    onClick={() => onClick(columnNumber)}>
+    <TableSearchHeaderCellDiv>
+      {[
+        `${headerText} `,
```

Comment:
You should make use of padding on the `<i>` element, rather than adding blank spaces.